lib/compat/wordpress-6.8/block-template-utils.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Make the title and description of the post format templates human readable.
+ *
+ * @param array $default_template_types Default template types.
+ * @return array Filtered template type data.
+ */
+function gutenberg_post_format_template_title_description( $default_template_types ) {
+	$post_formats = get_post_format_strings();
+
+	foreach ( $post_formats as $post_format_slug => $post_format_name ) {
+		$default_template_types[ 'taxonomy-post_format-post-format-' . $post_format_slug ] = array(
+			'title'       => sprintf(
+				/* translators: %s: Post format name. */
+				_x( 'Post Format: %s', 'Template name' ),
+				$post_format_name
+			),
+			'description' => sprintf(
+				/* translators: %s: Post format name. */
+				__( 'Displays the %s Post Format archive.' ),
```

Comment:
```suggestion
				__( 'Displays the %s post format archive.' ),
```

Nit: It was probably changed to lower case when committed in core.

See: https://github.com/WordPress/wordpress-develop/blob/de302c6c71697a3584c089a82fdee65be258fd9a/src/wp-includes/block-template-utils.php#L220